bentoml/frameworks/keras.py

```diff
@@ -197,24 +201,24 @@ def load(self, path):
                 open(self._custom_objects_path(path), 'rb')
             )
 
-        if self._store_as_json_and_weights:
-            # load keras model via json and weights if store_as_json_and_weights=True
-            self.create_session()
-            with self.graph.as_default():
-                with self.sess.as_default():
-                    with open(self._model_json_path(path), 'r') as json_file:
-                        model_json = json_file.read()
-                    model = keras_module.models.model_from_json(
-                        model_json, custom_objects=self._default_custom_objects
-                    )
-                    model.load_weights(self._model_weights_path(path))
-        else:
-            # otherwise, load keras model via standard load_model
-            model = keras_module.models.load_model(
-                self._model_file_path(path),
-                custom_objects=self._default_custom_objects,
-            )
-        return self.pack(model)
+        self.create_session()
+        with self.graph.as_default(), self.sess.as_default():
```

Comment:
We only need to set context to the current session only, since we will have the session bind to the default graph. This is fixed in the 1.0 branch on my end